src/Uno.UI.RemoteControl/Uno.UI.RemoteControl.csproj

```diff
@@ -43,6 +43,7 @@
 	  <UpToDateCheckInput Remove="HotReload\Messages\IMessage.cs" />
 	  <UpToDateCheckInput Remove="HotReload\Messages\XamlLoadError.cs" />
 	  <UpToDateCheckInput Remove="IRemoteControlProcessor.cs" />
+	  <UpToDateCheckInput Remove="IRemoteControlPreProcessor.cs" />
 	  <UpToDateCheckInput Remove="ProjectConfigurationAttribute.cs" />
 	  <UpToDateCheckInput Remove="RemoteControlClient.cs" />
```

Comment:
Why do we have all these? Doesn't look right to me

Reply:
I don't know why they're all there. Just copying what exists when adding something new. 
If this isn't appropriate and all these checks shouldn't be there, I'd like to argue that reviewing all `.csproj` files as a separate task/PR should be undertaken.

Reply:
Created https://github.com/unoplatform/uno/issues/12409 to tidy up after this PR is merged